public/player.js

```diff
@@ -35,32 +35,42 @@
     YT.ready(onYouTubeIframeAPIReady);
   };
 
-  function toggleHighlight(x) {
-    x.classList.toggle("highlighted-option")
-    setTimeout(function() {
-      x.classList.remove("highlighted-option")
-    }, 3000)
+  var funcCount = (function () {
+      var counter = 0;
+      return function () {return counter += 1;}
+  })();
+
+  function toggleHighlight(answer, arr, event) {
```

Comment:
Also I expect arr is not even an array. its a [NodeList](https://developer.mozilla.org/en/docs/Web/API/NodeList).
`AnswerFigureNodes` or something? is better name